server/internal/handler/commit_message.go

```
@@ -0,0 +1,410 @@
+		Model:           a.gitModel(),
+		SystemPrompt:    a.resolvedPrompt(repo),
+		MaxOutputTokens: draftMaxOutputTokens,
+		Messages: []pkgagent.Message{{
+			Role:    pkgagent.RoleUser,
+			Content: pkgagent.EncodeText(user.String()),
+		}},
+	})
+	if err != nil {
+		return "", false
+	}
+	defer stream.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `stream.Close` is not checked

(errcheck)